cmd/simulate.go

```diff
@@ -64,10 +64,8 @@ apps, mh acts on all apps in your mh config.`,
 
 func init() {
 	RootCmd.AddCommand(simulateCmd)
-	var setValuesFlag []string
 
-	simulateCmd.PersistentFlags().BoolP("printRendered", "p", false, "print rendered override values")
-	simulateCmd.PersistentFlags().StringSliceVar(&setValuesFlag, "set", nil,
+	simulateCmd.Flags().BoolVarP(&printRendered, "printRendered", "p", false, "print rendered override values")
```

Comment:
Persistent flags state... 

`A flag can be 'persistent' meaning that this flag will be available to the command it's assigned to as well as every command under that command. For global flags, assign a flag as a persistent flag on the root.`

Since multiple commands support printRendered perhaps its scope belongs on the root command which will then be picked up by these children commands, instead of duplicating this flag everywhere.

I should mention that I have no used, nor am I familiar with Cobra.  This just seems weird reading the docs.

Reply:
Yes @drekle, even initially I implemented the same, then realise `printRendered` is not used in all child commands, it is only for `apply` and `simulate`. Even for `destroy` `-p` means purge. But agree, that it is duplicate lines of code only.

Reply:
@prabdeb does it makes sense to add a intermediate command which supports printRendered, and then having apply and simulate inherit from that command instead of directly from root?